eng/Version.Details.xml

```diff
@@ -18,6 +18,11 @@
       <Sha>c5dd6a1da2e6d9b3423ab809fcda8af2927a408b</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.GenAPI" Version="8.0.0-beta.22524.5">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha>c5dd6a1da2e6d9b3423ab809fcda8af2927a408b</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true" />
```

Comment:
This SourceBuild attribute needs to be removed.  There can only be one specified per repo

Reply:
Ok. thanks. Updated and pushed again.